examples/scripts/minimal/ppo_bandit_rloo_large.py

```diff
@@ -0,0 +1,177 @@
+from collections import defaultdict
+
+import pandas as pd
+from datasets import load_dataset
+from rich.console import Console
+from rich.table import Table
+from transformers import (
+    AutoModelForCausalLM,
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    GenerationConfig,
+    HfArgumentParser,
+    PreTrainedModel,
+)
+
+from trl.trainer.ppov2_bandit_rloo_trainer import PPOConfig, PPOTrainer
+
+
+"""
+# run ppo
+python examples/scripts/minimal/ppo_bandit_rloo.py \
+    --learning_rate 3e-6 \
+    --output_dir models/minimal/ppo \
+    --per_device_train_batch_size 64 \
+    --gradient_accumulation_steps 1 \
+    --total_episodes 10000 \
+    --base_model EleutherAI/pythia-1b-deduped \
+    --non_eos_penalty \
+# run REINFORCE w/ RLOO; `--epochs 1 --nminibatches 1` in PPO is equivalent to REINFORCE
+accelerate launch --config_file examples/accelerate_configs/deepspeed_zero3.yaml \
+    examples/scripts/minimal/ppo_bandit_rloo_large.py \
+    --noptepochs 1 \
+    --nminibatches 1 \
+    --rloo_k 4 \
+    --learning_rate 3e-6 \
+    --output_dir models/minimal/ppo \
+    --per_device_train_batch_size 1 \
+    --gradient_accumulation_steps 16 \
+    --total_episodes 10000 \
+    --base_model mistralai/Mistral-7B-v0.1 \
```

Comment:
We probably should use a dataclass subclassing from https://github.com/huggingface/trl/blob/08655727487f8436980d4b538637832d1dcfa2e0/trl/trainer/model_config.py#L13